modules/test/playwright/helpers/HeadlessAdminUserApiHelper.ts

```diff
@@ -187,6 +217,51 @@ export class HeadlessAdminUserApiHelper {
 		);
 	}
 
+	async postOrganizationRoleUserAccountAssociation(
+		roleId: string,
+		userAccountId: string,
+		organizationId: string
+	) {
+		return this.apiHelpers.post(
+			`${this.apiHelpers.baseUrl}${this.basePath}/roles/${roleId}/association/user-account/${userAccountId}/organization/${organizationId}`
+		);
+	}
+
+	async postUserAccount(userAccount?: TUserAccount): Promise<TUserAccount> {
+		const userNumber = getRandomInt();
+
+		userAccount = await this.apiHelpers.post(
+			`${this.apiHelpers.baseUrl}${this.basePath}/user-accounts`,
+			{
+				data: {
+					alternateName: 'User' + userNumber,
+					emailAddress: 'User' + userNumber + '@liferay.com',
+					familyName: 'User' + userNumber,
+					givenName: 'User' + userNumber,
+					...(userAccount || {}),
+				},
```

Comment:
please add also failOnStatusCode: true -> this will make the test failing if the request fails